repository.go

```diff
@@ -139,16 +137,8 @@ func (r *Repository) Hydrate(client *Client) error {
 		}
 	}
 
-	if r.Tags == nil {
-		r.Tags = &TagConnection{}
-	}
-	if r.Tags.PageInfo.HasNextPage {
```

Comment:
hmm this is going to present a problem for the CLI when doing `opslevel list repositories -o json`

Ugh man this is not a facet we considered 😮‍💨